New Chapter in Community Handbook on Drop-in Sessions (#3694)
the-turing-way/the-turing-way: book/website/community-handbook/coworking/coworking-dropins.md

(ch-coworking-dropins)=

# Drop-in Sessions

Drop-in sessions are an informal collaboration format where a team or group of people host regular opportunities for other people within the community or organisation to ask questions or start a conversation on a topic of their interest.

These sessions typically include an online component, typically using [Zoom](https://www.zoom.us/). 

> They may follow a different format or sign up person, depending on the needs and preferences of the organising team and the target community the sessions are designed for.

Reviewer: ```suggestion
They may follow a different format, depending on the needs and preferences of the organising team and the target community the sessions are designed for.
```
Or please explain what signup person means?

Author: Thank you, it was a typo!

The primary goal of these sessions is to provide an opportunity for answering targeted questions on a subject that the team holds expertise in, such as data protection, stakeholder engagement, or research software engineering.
The sessions are a resource for community or organisation members to ask questions, receive targeted advice and support, and create space to start co-working or futher collaboration.
The sessions are also an opportunity to learn about and connect with new teams and individuals that an attendee may not regularly connect with.

```{figure} ../../figures/research-communities.*
---
height: 400px
name: research-communities

> alt: Cartoon-like sketch of people in different nodes connecting into a central node with a robot in the middle. The people represent researchers in different research domains. One group is building a rocket ship, another group is fixing a car, and another group is working on a hot air balloon. In the central node, researchers are performing different tasks to build a robot. Some of researchers are taking the findings from the central node to the other nodes.

Reviewer: ```suggestion
alt: Cartoon-like sketch of people in different nodes connecting into a central node with a robot in the middle. The people represent researchers in different research domains. One group is building a rocket ship, another group is fixing a car, and another group is working on a hot air balloon. In the central node, researchers are performing different tasks to build a robot. Some of the researchers are taking the findings from the central node to the other nodes.
```

---
_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
```

## Attending a drop-in session

There are many ways to manage attendee sign ups that occur before the call and during the call.
The best method will depend on the size of your team, the desired audience, and the expected topics.

> Sign ups are a way to approach scheduling different attendee requests

Reviewer: ```suggestion
Sign ups are a way to set up the schedule based on the requirements of attendees.
```

Here we share a few approaches, ranging from low touch to high touch:
- Attendees drop in at any point during a session. This provides them flexibility and removes the need for prior scheduling, but may require them to sit in on other conversations and wait to have their question answered.
- Allow attendees to sign up directly to fill open slots.
- Attendees directly message hosts who schedule them into open slots. 

When there are more requests than available slots, any additional requests can be accommdated through multiple breakout rooms (if there are more hosts available) or scheduled for a future meeting.

## Resources for drop-in sessions

The resources used for these sessions help facilitate coordination amongst the organisers and also to enable attendee sign up.

1. Online call by [Zoom](https://www.zoom.us/): The Zoom link is shared on an internal Turing webpage and on Slack.

> 2. In person location (optional): A reserved meeting room or location at a shared organisation space

Reviewer: ```suggestion
2. In person location (optional): A reserved meeting room or location at a shared organisation space.
```

> 3. Online github repo: share information about the session, including [the schedule](https://github.com/alan-turing-institute/infrastructure-drop-ins/wiki/Schedule) and [how to sign up](https://github.com/alan-turing-institute/infrastructure-drop-ins)

Reviewer: ```suggestion
3. Online GitHub repo: share information about the session, including [the schedule](https://github.com/alan-turing-institute/infrastructure-drop-ins/wiki/Schedule) and [how to sign up](https://github.com/alan-turing-institute/infrastructure-drop-ins).
```

> 4. Engagement and impact tracking resource: Sheet with information on the topics raised, next steps, and the overall impact of the session

Reviewer: ```suggestion
4. Engagement and impact tracking resource: Sheet with information on the topics raised, next steps, and the overall impact of the session.
```

- Session hosts can coordinate amongst each other to schedule and assign slots during the session

### During the Session

- Make sure there is someone monitoring the main Zoom room and physical space to welcome new attendees
- It is especially important that someone is available before each slot to guide attendees to a breakout room, if those are being used 

### After the Session

Record notes for next steps and follow-ups in a shared document so that teammates can pick up conversations and collaborations in future sessions.

## Case Study: Turing Infrastructure Drop-In Sessions

At the Turing Institute, there are a few different variations of drop-in sessions including the [Turing Infrastructure Drop-in Sessions](https://github.com/alan-turing-institute/infrastructure-drop-ins) (online and in person), the Turing Research Ethics (TREx) Virtual Drop in Session (online only), the Turing Skills Drop in Session (online and in person), and the Data Protection virtual drop in (online only).

This section will highlight the Turing Infrastructure Drop-In Sessions as a case study to illustrate how a drop-in session can run in practice.

> ### Attending a drop-in session

Reviewer: Starting from here it becomes less informative compared to before - the information you share here is already available above. So I would agree with Arielle that a case study format would make more sense.

Author: Thanks for the feedback, @EstherPlomp! Originally the content was interlinked and I think it is much better split up like this. Do you think it's worth including the case study as an example (as written), or is it too repetitive? The primary value I see is sharing the public github link for the Turing sessions which could help "ground" the chapter.

Another approach could be to change the "Case Study" section to a "List of Examples" section.

Reviewer: I do like it as a case study so I would still include it! But then indeed more as a case study or List of examples to clarify that it may not contain super new information!

> # Drop-in Sessions

Reviewer 2: Do we need chapter tags still? Check w/ infra group!

Reviewer 3: They've never been "needed" (as in, the book still builds without them) but it's still good practice (they're needed if anyone ever wants to crosslink to that page) -
https://book.the-turing-way.org/community-handbook/style/style-crossref

> Drop-in sessions are an opportunity for a team or group of people to host regular opportunities for other people within the community or organisation to ask questions or start a conversation on a topic of their interest.

Reviewer 2: ```suggestion
Drop-in sessions are an informal collaboration format where a team or group of people host regular opportunities for other people within the community or organisation to ask questions or start a conversation on a topic of their interest.
```

double use of opportunities otherwise

> At the Turing, these include the [Turing Infrastructure Drop-in Sessions](https://github.com/alan-turing-institute/infrastructure-drop-ins) (online and in person), the TREx Virtual Drop in Session (online only), the Turing Research Ethics (TREx) Virtual Drop in Session (online only), and the Data Protection virtual drop in (online only).

Reviewer 2: ```suggestion
At the Turing, these include the [Turing Infrastructure Drop-in Sessions](https://github.com/alan-turing-institute/infrastructure-drop-ins) (online and in person), the Turing Research Ethics (TREx) Virtual Drop in Session (online only), the Turing Skills Drop in Session (online and in person), and the Data Protection virtual drop in (online only).
```

Reviewer 2: duplication of a session title so I've added another drop in!

These sessions typically include an online component, typically using [Zoom](https://www.zoom.us/). 

> They may follow a different format or sign up person, depending on the needs and preferences of the organising team and the target community the sessions are designed for.

Reviewer 2: ```suggestion
They may follow a different format or sign up person, depending on the needs and preferences of the organising team and the target community the sessions are designed for.
```
This sentence is a little confusing - maybe something like "The format, or method of signing up for drop ins may vary depending on.... "?

> The primary goal of these sessions is to provide an opportunity for answering targeted questions on a subject that the team holds expertise in, such as data protection or research software engineering.

Reviewer 2: I think this paragraph could got above the section what drop ins are at Turing

The sessions are a resource for community or organisation members to ask questions, receive targeted advice and support, and create space to start co-working or futher collaboration.
The sessions are also an opportunity to learn about and connect with new teams and individuals that an attendee may not regularly connect with.

```{figure} ../../figures/research-communities.*
---
height: 400px
name: research-communities
alt: Cartoon-like sketch of people in different nodes connecting into a central node with a robot in the middle. The people represent researchers in different research domains. One group is building a rocket ship, another group is fixing a car, and another group is working on a hot air balloon. In the central node, researchers are performing different tasks to build a robot. Some of researchers are taking the findings from the central node to the other nodes.
---
_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
```

> ## Attending a drop-in session

Reviewer 2: I think we might want to add in a bit here on what people will want to take into account when setting up a drop in session - how to promote to desired audiences, what the concerns or barriers might be to attending, how to address them etc?

Reviewer 2: Okay coming back to this - I'm wondering if we put all the generic advice up top then add a "case study" for the Turing infrastructure drop ins? Not a hill for me but think it might make a bit more sense?

Author: Yeah coming back to it this morning, I think you're right! 

Will keep the generic stuff up to and add a case study at bottom. I'll focus on the infra drop-ins because we have more public resources for that

The schedule also allows interested attendees to reference when to attend, based on the individual or team that is hosting on a particular week.

In order to sign up, an interested attendee will directly message the hosts for that session on the organisation's collaboration platforms, such as Slack, Microsoft Teams, or e-mail.
The hosts will then assign the attendee a 30-60 minute slot.

## Resources for drop-in sessions

The resources used for these sessions help facilitate coordination amongst the organisers and also to enable attendee sign up.

1. Online call by [Zoom](https://www.zoom.us/): The Zoom link is shared on an internal Turing webpage and on Slack.
2. In person location (optional): A reserved meeting room or location at a shared organisation space
3. Online github repo: share information about the session, including [the schedule](https://github.com/alan-turing-institute/infrastructure-drop-ins/wiki/Schedule) and [how to sign up](https://github.com/alan-turing-institute/infrastructure-drop-ins)
4. Engagement and impact tracking resource: Sheet with information on the topics raised, next steps, and the overall impact of the session

> These sessions are not recorded. 

Reviewer 2: I think we should share reasoning here on why they're not recorded

## Format of the session

This format is drawn from the [Turing Infrastructure Drop-in Sessions](https://github.com/alan-turing-institute/infrastructure-drop-ins) which uses the following format:

| Duration | Activity |
| ---- | -------- |
| 5 mins | 👋 Welcome attendees and assign slot |
| 30 mins | :speech_balloon: Slot 1 |
| 30 mins | :speech_balloon: Slot 2 |
| 30 mins | :speech_balloon: Slot 3 |
| 30 mins | :speech_balloon: Slot 4 |
| 5 mins | 👋 Close session |

Each slot may run longer or shorter, but the standard length is 30 minutes.
Multiple slots may run in parallel through breakout rooms.

## Chairing drop-in sessions

### Before the Session

- Session hosts can double check the schedule to make sure the correct information is documented for that session
    - Update host names and affiliations
    - Correct Zoom link and physical location of the session
    - Share session on Slack and other channels inviting attendees to the event
- Session hosts can coordinate amongst each other to schedule and assign slots during the session

#### Approaches to Sign ups

There are many ways to manage sign ups that occur before the call and during the call.

> The best method will depend on the size of your team.

Reviewer 2: ```suggestion
The best method will depend on the size of your team, the desired audience, and the expected topics.
```

Sign ups are a way to approach scheduling different attendee requests
Here we share a few approaches, ranging from low touch to high touch:

> - Attendees drop in at any point during a session. This may require them to sit on other conversations and wait to have their question answered.

Reviewer 2: ```suggestion
- Attendees drop in at any point during a session. This may require them to sit in on other conversations and wait to have their question answered.
```

- Allow attendees to sign up directly to fill open slots.
- Attendees directly message hosts who schedule them into open slots. 

When there are more requests than available slots, any additional requests can be accommdated through multiple breakout rooms (if there are more hosts available) or scheduled for a future meeting.

### During the Session

- Make sure there is someone monitoring the main Zoom room and physical space to welcome new attendees
   - It is especially important that someone is available before each slot to guide attendees to a breakout room, if those are being used 

### After the Session

> Record notes for next steps and follow-ups in a shared document so that teammates can pick up conversations and collaborations in future sessions.

Reviewer 2: Noticed that bullets are a bit inconsistent between headings
